packages/devtools_app/lib/src/screens/performance/performance_screen.dart

```diff
@@ -164,7 +164,8 @@ class PerformanceScreenBodyState extends State<PerformanceScreenBody>
             },
           ),
         Expanded(
-          child: FeatureFlags.embeddedPerfetto
+          child: FeatureFlags.embeddedPerfetto &&
```

Comment:
can `useLegacyTraceViewer change here? Should we perhaps add a listenable builder to ensure a refresh
?

Reply:
done